config/migrations/clickhouse/0_initial_clickhouse.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS flipt_counter_analytics (
+    `timestamp` DateTime('UTC'), `analytic_name` String, `namespace_key` String, `flag_key` String, `flag_type` Enum('VARIANT_FLAG_TYPE' = 1, 'BOOLEAN_FLAG_TYPE' = 2), `reason` Enum('UNKNOWN_EVALUATION_REASON' = 1, 'FLAG_DISABLED_EVALUATION_REASON' = 2, 'MATCH_EVALUATION_REASON' = 3, 'DEFAULT_EVALUATION_REASON' = 4), `match` Bool, `value` UInt32
```

Comment:
Your PR is already big enough, but I think we should make an issue to get an IT done, if there isn't an issue for that already.

One thing I think that could be easy to miss is extending these enums in the future.
If we fail to update this accordingly, we could cause failures to insert into CH.
Perhaps we can come up with a test case that will ensure that doesn't happen.

Reply:
@GeorgeMac Yup will create an issue for this.